main/tests/server/src/com/google/refine/importers/ExcelImporterTests.java

```diff
@@ -266,41 +235,28 @@ public void readXlsxAsText() throws IOException {
             Assert.fail(e.getMessage());
         }
 
-        Assert.assertEquals(project.rows.size(), ROWS);
-        Assert.assertEquals(project.rows.get(1).cells.size(), COLUMNS);
-
-        final NumberFormat numberFormat = DecimalFormat.getInstance(Locale.getDefault());
-        Assert.assertEquals((String) project.rows.get(1).getCellValue(0), numberFormat.format(1.1));
-        Assert.assertEquals((String) project.rows.get(2).getCellValue(0), numberFormat.format(2.2));
-
-        assertEquals((String) project.rows.get(1).getCellValue(1), "FALSE");
-        assertEquals((String) project.rows.get(2).getCellValue(1), "TRUE");
-
-        // Skip col 2 where old Calendar test was
-        assertEquals((String) project.rows.get(1).getCellValue(3), NOW_STRING); // Date
-
-        assertEquals((String) project.rows.get(1).getCellValue(4), " Row 1 Col 5");
-        assertEquals((String) project.rows.get(1).getCellValue(5), "");
-
-        assertEquals((String) project.rows.get(1).getCellValue(6), "1");
-        assertEquals((String) project.rows.get(2).getCellValue(6), "2");
-
-        assertEquals(project.rows.get(1).getCellValue(7), "1");
-        assertEquals(project.rows.get(2).getCellValue(7), "2");
-
         final DecimalFormat decimalFormat = (DecimalFormat) NumberFormat.getNumberInstance(Locale.getDefault());
         decimalFormat.applyPattern("0.00");
-        assertEquals(project.rows.get(1).getCellValue(8), decimalFormat.format(100.00) + "%");
-        assertEquals(project.rows.get(2).getCellValue(8), decimalFormat.format(200.00) + "%");
-
-        assertEquals(project.rows.get(1).getCellValue(9), "0001");
-        assertEquals(project.rows.get(2).getCellValue(9), "0002");
-
-        assertEquals(project.rows.get(ROWS - 1).getCellValue(10), "(617) 235-1322");
-
-        assertEquals(project.rows.get(ROWS - 1).getCellValue(11), NOW_STRING.substring(0, 10)); // date only
-
-        assertEquals(project.rows.get(ROWS - 1).getCellValue(12), "$" + numberFormat.format(1234.56));
+        final NumberFormat numberFormat = DecimalFormat.getInstance(Locale.getDefault());
```

Comment:
Not really related to this PR, but I think it would be better to choose a non-US locale and pin it rather than allowing the tests to vary based on where they are run.

Reply:
I have thought about proposing that indeed… One downside I would see with it is that it would let us bake localized strings into test assertions, which would mean the test suite could break if a translator decides to change the translation to something else.
In any case, moving to something like that would have impacts in quite a few other places in the test suite I expect, so better leave that for another PR.